server/src/services/organizations.service.ts

```diff
@@ -166,6 +170,10 @@ export class OrganizationsService {
           qb.andWhere('lower(user.lastName) like :lastName', {
             lastName: `%${options?.lastName.toLowerCase()}%`,
           });
+        if (options?.status)
+          qb.andWhere('lower(organization_user.status) like :status', {
```

Comment:
Should not be wild card search, should be 
```
qb.andWhere('organization_user.status = :status'
```
no need to convert into lower cases